### ISSUE TITLE
Setting cookies?

### ISSUE THREAD

**Issue by volkanogretmen** [stevenvachon/broken-link-checker#122]
Hi guys,

I've been trying to set cookies with BLC, but didn't succeed yet. Therefore, BLC is getting "blocked" by a Cookie Wall and cannot proceed. Normally, when using `?policy=accepted` behind the URL, it would automatically "accept" the cookies, but that doesn't seem to work either, because it gets redirected to a `301 Moved Permanently` and doesn't proceed from there.

Any ideas how to deal with this?

Thanks!

Volkan

**Comment by volkanogretmen** [comment]
That's too bad. I think BLC would really benefit from the possibility of adding custom headers as it's quite a common feature that people tend to use. I hope it will be added soon. Thanks a lot for your response.

**Comment by stevenvachon** [comment]
I agree. I just can't comment on when or which version it will be implemented.
